reg/correctMotion_DFT.m

```diff
@@ -0,0 +1,64 @@
+function correctMotion_DFT(hdf5_dir,hdf5_in_name,hdf5_out_name)
+
+dataset = '/Data/Images';
+
+h5att = h5info(fullfile(hdf5_dir,hdf5_in_name),dataset);
```

Comment:
In your previous functions (like `concatenateHDF5`) I found myself padding with various `pwd`s for the directory input.

In general, I think it's nice to minimize the number of function parameters.